frontend/server/src/Controllers/TeamsGroup.php

```diff
@@ -333,4 +334,23 @@ public static function apiRemoveTeam(\OmegaUp\Request $r): array {
 
         return ['status' => 'ok'];
     }
+
+    /**
+     * Gets a list of teams groups. This returns an array instead of an object
+     * since it is used by typeahead.
+     *
+     * @omegaup-request-param null|string $query
+     *
+     * @return array{results: list<ListItem>}
+     */
+    public static function apiList(\OmegaUp\Request $r): array {
+        // Authenticate user
+        $r->ensureMainUserIdentity();
+
+        $query = $r->ensureString('query');
+
+        return [
+            'results' => \OmegaUp\DAO\TeamGroups::findByNameOrAlias($query),
+        ];
```

Comment:
```suggestion
        return \OmegaUp\DAO\TeamGroups::findByNameOrAlias($query);
```

la intención es usar esta API en typescript? si sí, hay que hacerle caso al comentario en L339. si no, hay que dejar esto como está y quitar parte del comentario de L339-L340.

Reply:
Si, sólo se usaría con esa finalidad.

Reply:
ah perdón, no typescript sino typeahead!